src/crimson/osd/shard_services.cc

```diff
@@ -618,7 +618,15 @@ seastar::future<Ref<PG>> ShardServices::handle_pg_create_info(
       return get_map(info->epoch).then(
 	[&info, FNAME, this](cached_map_t startmap)
 	-> seastar::future<std::tuple<Ref<PG>, cached_map_t>> {
+	  LOG_PREFIX(ShardServices::handle_pg_create_info);
```

Comment:
This function already declared the LOG_PREFIX (l.613). Can you please correct the prefix to have the function name? (not trim_maps)
`DEBUG();` can be used here instead.

Reply:
Yes, will correct it.

By the way, I've been wondering, for a long time, whether capturing `FNAME` by value will involve unnecessary cpu overhead, which is also why I defined the prefix in every seastar continuation in all my PRs whenever logs are needed instead of capturing `FNAME`. What do you think?

Reply:
We may need to improve that, especially when debug is off

Reply:
I modified the PR a bit, please take a look again. Thanks.

Reply:
For now, just capture FNAME.  It's a const char *, so I really doubt the overhead is at all measurable.

Reply:
> For now, just capture FNAME and leave a single LOG_PREFIX at the start of the method -- I don't want it duplicated within a method/function. It's a const char *, so I really doubt the overhead is at all measurable.

Ah, yes.